main/ui/src/main/java/org/cryptomator/ui/util/mount/LinuxGvfsWebDavMounter.java

```diff
@@ -70,4 +74,11 @@ public void unmount() throws CommandFailedException {
 		};
 	}
 
+	private Script openFMWithWebdavSchema(String schemaName, URI uri){
+		return Script.fromLines(
+				"set -x",
+				"xdg-open \""+schemaName+":$DAV_SSP\"")
```

Comment:
I have added the $DAV_SCHEME and openFMWithWebdavSchema accepts only one paramter now.